package.json

```diff
@@ -142,58 +142,58 @@
     "use-sync-external-store": "1.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.18.10",
+    "@babel/core": "^7.18.13",
     "@babel/helper-module-imports": "^7.18.6",
     "@babel/plugin-transform-react-jsx": "^7.18.10",
     "@babel/plugin-transform-typescript": "^7.18.12",
     "@babel/preset-env": "^7.18.10",
-    "@babel/types": "^7.18.10",
+    "@babel/types": "^7.18.13",
     "@redux-devtools/extension": "^3.2.3",
     "@rollup/plugin-alias": "^3.1.9",
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-replace": "^4.0.0",
-    "@rollup/plugin-typescript": "^8.3.4",
-    "@swc/core": "^1.2.239",
+    "@rollup/plugin-typescript": "^8.4.0",
+    "@swc/core": "1.2.242",
```

Comment:
Updating to v1.2.244 causes tests to break.
https://github.com/swc-project/swc/blob/main/CHANGELOG.md#12244---2022-08-26
Not sure what change makes so...